diesel/src/deserialize.rs

```diff
@@ -20,7 +20,7 @@ pub type Result<T> = result::Result<T, Box<dyn Error + Send + Sync>>;
 /// trait is to convert from a tuple of Rust values that have been deserialized
 /// into your struct.
 ///
-/// This trait can be [derived](derive.Queryable.html)
+/// This trait can be [derived](Queryable)
```

Comment:
```suggestion
/// This trait can be [derived](derive@Queryable)
```

See the [documentation](https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html) for details about the syntax.